roles/container-engine/containerd/defaults/main.yml

```diff
@@ -65,6 +65,7 @@ containerd_default_runtime:
 #     type: io.containerd.kata.v2
 #     engine: ""
 #     root: ""
+#     privileged_without_host_devices: true
```

Comment:
According to config.toml.j2 in this PR, the default value seems false.
Which do we need?

Reply:
@oomichi you're right, it's default value is `false`. The idea of this patch is to allow Kubespray's users to change it to `true`

Reply:
Ah, OK.
I was confused with the difference because the above is defaults/main.yml which should show the default values.
But the above lines are for additional configuration example.
It is fine for me.

/lgtm